xarray/core/dataset.py

```diff
@@ -2742,6 +2882,37 @@ def head(
             The keyword arguments form of ``indexers``.
             One of indexers or indexers_kwargs must be provided.
 
+        Examples
+        --------
+        >>> dataset = xr.Dataset(
```

Comment:
This example is valid xarray but I think we can come up with something that better motivates why a user might want to use the `.head` method. In particular they would probably use it when the first values in their dataset are also the most important.

Can we think of a dataset that has that property? How about financial transactions sorted by profit? We could use `dataset.sortby` first then `.head` to get the biggest sales. If we want to motivate using dataset.head rather than just `dataarray.head` we would also need some other variables, so how about a dataset that had e.g. `[festival_name, profit]` as data variables, as a function of a time coordinate? Then you could sortby profit and call head.